Upload A File/Download Upload Library.gcomp/subVI/Upload a File.gviweb

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<SourceFile Checksum="F7932AC1E7C40BF2787DC5DE31368B42A1DA23509331B2D1B34BFBF258EB8A3EE4A5C47EC64E1860AC5DC0F23A41C9246947CFAC3F4C90930B3E9C937BC203AA" Timestamp="1D5308A57895F92" xmlns="http://www.ni.com/PlatformFramework">
+<SourceFile Checksum="77D530C3FF6BC49E1FD0DC2801855FE888F47FE9A9D8DAC90DCA488D2D3DEC61876E831F84FA01F4AE55FEE83640DBB3ACA013CAD539D90FC111FEAAA15A2221" Timestamp="1D53EA96F2960E3" xmlns="http://www.ni.com/PlatformFramework">
```

Comment:
Upload A File got a lot simpler :P

![image](https://user-images.githubusercontent.com/1588923/61586245-cc8db980-ab34-11e9-82f6-4f2c074f8877.png)

Could be neat in the future to add more capabilities that the `<input>`  element supports like a file types filter, multi-file select, etc.